src/components/Header/TopNav/LogoLink.js

```diff
@@ -22,40 +18,45 @@ const StyledLink = styled(Link)`
     width: ${remcalc(54)};
   }
 `
+const HomepageLogo = styled.img`
+  width: 49px;
+  height: 36px;
+  margin-top: ${remcalc(6)};
+`
+
+const LogoLink = ({ slug, isServicePage, isSpecialityPage, isHomePage }) => {
+  const renderSvg = isServicePage || isSpecialityPage || isHomePage
+
+  const fillColorInitial = logoColors[slug] || logoColors['default']
```

Comment:
`logoColors[slug || 'default']` ; only removes one word but every little helps!

Reply:
@GBouffard that way it wouldn't fallback to `default` it `slug` is defined, even if it's not mapped on `logoColors`

Reply:
Good point. I didn't think of this part. Thanks Fabio